numbers/randBeta.go

```diff
@@ -0,0 +1,182 @@
+package numbers
+
+import (
+	"math"
+	"math/rand"
+	"log"
+)
+
+const RelError float64 = 1.0e-8
```

Comment:
Are these constants specific to one function?  I would put them inside that function.  Right now they are kind of general names and available to all functions inside "numbers".